it/controllers/DeleteVendorStateControllerISpec.scala

```diff
@@ -28,13 +28,27 @@ import support.IntegrationBaseSpec
 
 class DeleteVendorStateControllerISpec extends IntegrationBaseSpec {
 
-  "Invoking the delete vendor state endpoint" should {
+  "Calling the delete vendor state endpoint" should {
     "return a 204 status code" when {
       "a valid request is made" in new Test {
 
         override def setupStubs(): StubMapping = {
           AuthStub.authorised()
-          DownstreamStub.onSuccess(DownstreamStub.DELETE, stubUri, Map.empty, NO_CONTENT, JsObject.empty)
+          DownstreamStub.onSuccess(DownstreamStub.DELETE, downstreamUri, downstreamQueryParams, NO_CONTENT, JsObject.empty)
+        }
```

Comment:
Since `downstreamUri` and `downstreamQueryParams` are defined in `Test` you can make this the default implementation of `setupStubs` in the `Test` class and remove this.